### DISCUSSION TITLE
添加百度网盘时遇到了一些问题，麻烦各位进来看看，非常感谢！！！

### DISCUSSION THREAD

**Discussion by jonnycat2018** [AlistGo/alist#7373] [Q&A] [answered]
### 添加百度网盘时遇到了一些问题，麻烦各位进来看看，非常感谢！！！
环境如下：
平台：软路由;
AList 版本: v3.38.0;
框架:debian上的 Docker搭建的 AList;
操作方法：在添加百度网盘时，已选中Web 代理 。
WebDAV 策略使用本地代理。
下载接口选用非官方，已开通 SVIP。
通过 WebUI 进入 AList 访问到的百度网盘，视频文件无法播放。
下载文件时，会下载一个与源文件同名的.JSON文件，内容如下:
****{"code":500,"message":"http request [https://d.pcs.baidu.com/file/afc3b79d5jb764c68f0a33b6c4c22359?fid=154160693-250528-29064072290615\u0026rt=pr\u0026sign=FDtAERK-DCb740ccc5511e5e8fedcff06b081203-iLvE1zIIx2xKN%2brw6fAaj8PavS4%3d\u0026expires=8h\u0026chkbd=0\u0026chkv=0\u0026dp-logid=9163074532007183189\u0026dp-callid=0\u0026dstime=1729583579\u0026r=986858325\u0026vuk=154160693\u0026origin=dlna\u0026clienttype=0] failure,status: 403 response:{\"error_code\":31326,\"error_msg\":\"user is not authorized, hitcode:106\",\"request_id\":9163074641652169479}","data":null}****
使用官方接口也无法播放和下载，提示如下:
****{"code":500,"message":"Get \"\": unsupported protocol scheme \"\"","data":null}****
以下是部分日志内容 如下：
****[GIN] 2024/10/22 - 07:15:37 | 200 |  324.195448ms |     192.168.9.1 | GET      "/d/天痕-百度网盘/[工ZX胞 1]【中配版】[ ]/01.mp4?sign="
[31mERRO[0m[2024-10-22 07:18:37] http request [https://d.pcs.baidu.com/file/5eb0558e9qa67d4fcf5369fde654588e?fid=154160693-250528-454311918047073&rt=pr&sign=FDtAERK-DCb740ccc5511e5e8fedcff06b081203-t3fEQLAYxG7A9lE1tOA4tX9sW2g%3d&expires=8h&chkbd=0&chkv=0&dp-logid=9162521298920051710&dp-callid=0&dstime=1729581518&r=725785006&vuk=154160693&origin=dlna&clienttype=0] failure,status: 403 response:{"error_code":31326,"error_msg":"user is not authorized, hitcode:106","request_id":9162521422192882167} 
[GIN] 2024/10/22 - 07:18:37 | 200 |  657.576263ms |     192.168.9.1 | GET      "/d/天痕-百度网盘/[工ZX胞 1]【中配版】[ ]/02.mp4?sign="
[31mERRO[0m[2024-10-22 07:18:38] http request [https://d.pcs.baidu.com/file/5eb0558e9qa67d4fcf5369fde654588e?fid=154160693-250528-454311918047073&rt=pr&sign=FDtAERK-DCb740ccc5511e5e8fedcff06b081203-t3fEQLAYxG7A9lE1tOA4tX9sW2g%3d&expires=8h&chkbd=0&chkv=0&dp-logid=9162521457847131208&dp-callid=0&dstime=1729581518&r=739294304&vuk=154160693&origin=dlna&clienttype=0] failure,status: 403 response:{"error_code":31326,"error_msg":"user is not authorized, hitcode:106","request_id":9162521570637286473} ****
麻烦帮我看看这是什么问题，非常感谢。

**Comment by jonnycat2018** [comment, marked as answer]
已经解决，确认是百度账号问题。

**Reply by outsider1997** [reply]
请问账号问题是怎么解决的

**Reply by jonnycat2018** [reply]
> 请问账号问题是怎么解决的

使用百度开放平台找百度客服解决的。

**Reply by Rayciu** [reply]
> > 请问账号问题是怎么解决的
> 
> 使用百度开放平台找百度客服解决的。

大佬，请问是怎么去跟百度客服说啊?，我也是开完SVIP 之后，突然就看不了视频了，我用没开 VIP 的百度网盘账号又能看视频，搞一天了还没解决，把我整麻了
